examples/04_glm_first_level/plot_design_matrix.py

```diff
@@ -114,6 +114,27 @@
 plt.show()
 
 
+# %%
+# Correlation between regressors
+# ------------------------------
+# We can plot the correlation between the regressors of our design matrix.
+# This is important to check as highly correlated regressors can affect
+# the effficieny of
+# `your design https://imaging.mrc-cbu.cam.ac.uk/imaging/DesignEfficiency#Correlation_between_regressors`_. # noqa
+#
+
+from nilearn.plotting import plot_design_matrix_correlation
+
+fig3, (ax1, ax2, ax3) = plt.subplots(figsize=(15, 5), nrows=1, ncols=3)
+plot_design_matrix_correlation(X1, axes=ax1)
```

Comment:
while working on this I noticed we have 2 functions in the entire code base that use
`ax` instead of `axes` in their function definition.

- plot_design_matrix
- plot_contrast_matrix

```
$ grep -nr " ax=" nilearn/**/*.py

nilearn/plotting/matrix_plotting.py:339:    contrast_def, design_matrix, colorbar=False, ax=None, output_file=None
nilearn/plotting/matrix_plotting.py:459:def plot_design_matrix(design_matrix, rescale=True, ax=None, output_file=None):
```

I would suggest make those consistent with the rest of the codebase.